sqllineage/drawing.py

```diff
@@ -152,7 +158,9 @@ def lineage(payload):
 
     req_args = Namespace(**payload)
     sql = extract_sql_from_args(req_args)
-    lr = LineageRunner(sql, verbose=True)
+    dialect = getattr(req_args, "dialect", DEFAULT_DIALECT)
+    use_sql_parse = bool(getattr(req_args, "use_sqlfluff", DEFAULT_USE_SQLFLUFF))
+    lr = LineageRunner(sql, verbose=True, dialect=dialect, use_sqlparse=use_sql_parse)
```

Comment:
For Web UI lineage visualization, it's all forced to use SQLFLUFF as parser, correct? As payload here is passed from frontend and I don't see any frontend code update.

Later if we decide to only keep sqlfluff, it could be a good idea to add dialect option in frontend UI.

Reply:
I have tried to put some controls in the UI to select some dialects and use `sqlfluff` as the parser, but I don't know what is missing to make it work properly.

Reply:
This is lower priority for the moment. I can take a look later.

Reply:
Frontend code has been updated, together with draw.py. I merged the two settings (use_sqlfluff and dialect) into one unified dialect setting, by introducing a pseudo dialect called "non-validating" used for sqlparse only.

I'm still thinking about whether we should do the same on backend, by removing the use_sqlfluff option in LineageRunner. Let me know your thought.